pkgs/data/misc/colemak-dh/default.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "colemak-dh";
+  version = "0.pre+date=2022-08-07";
```

Please change the version to match the manual (see [here](https://github.com/NixOS/nixpkgs/pull/184860#discussion_r943147750) if you were going off RFC 107).

```suggestion
  version = "unstable-2022-08-07";
```